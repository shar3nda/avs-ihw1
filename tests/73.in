7
5 4 7 8 2 7 10