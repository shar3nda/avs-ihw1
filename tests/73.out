0 6 9 3 3 7 3 3 -1